docs/plans/benchmark-improvements/006-update-benchmarking-documentation.md

```
@@ -0,0 +1,230 @@
+
+```typescript
+bench.add('name', async () => {
+  await asyncOp()
+})
+```
+
+### Update Output Format
+
+tinybench uses `console.table(bench.table())` which outputs a different format:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)